octavia_f5/restclient/as3objects/irule.py

```diff
@@ -44,46 +44,38 @@
     HTTP::header remove "X-Forwarded-Proto"
     HTTP::header insert "X-Forwarded-Proto" $client_protocol
 }"""
-X_SSL_CLIENT_VERIFY = """when CLIENTSSL_CLIENTCERT {
+X_SSL_CLIENT_VERIFY = """when HTTP_REQUEST {
+    if { [HTTP::has_responded] }{ return }
     if { [SSL::cert count] > 0 }{
         set verify_result [SSL::verify_result]
     }
-}
-when HTTP_REQUEST {
-    if { [HTTP::has_responded] }{ return }
     if { [info exists verify_result] } {
         HTTP::header insert "X-SSL-Client-Verify" $verify_result
     }
 }"""
-X_SSL_CLIENT_HAS_CERT = """when CLIENTSSL_CLIENTCERT {
-  set cert_count [SSL::cert count]
-}
-when HTTP_REQUEST {
+X_SSL_CLIENT_HAS_CERT = """when HTTP_REQUEST {
     if { [HTTP::has_responded] }{ return }
+    set cert_count [SSL::cert count]
     if { [info exists cert_count] && $cert_count > 0 }{
```

Comment:
Ca we just use `if { [SSL::cert count] > 0 }{` since it's already used in other iRules this way?

Reply:
Yes, makes sense to remove it.